src/model/error.messages.ts

```diff
@@ -14,6 +14,7 @@ export enum ErrorMessages {
   ERROR_404 =  "Page not found",
   ERROR_500 = "An internal server error has occurred",
   ERROR_SUMMARY_TITLE = "There is a problem",
+  ERROR_TEXT_BOX_EMPTY = "Enter a reason for your objection",
```

Comment:
Can this be renamed to something like `ERROR_REASON_EMPTY` please?

Reply:
Thanks Tori! Fix done.